### DISCUSSION TITLE
iOS app?

### DISCUSSION THREAD

**Discussion by Nephilimi** [izivkov/GShockTimeServer#3] [Q&A] [answered]
Any chance of a iOS version of this project?

**Comment by izivkov** [comment]
You mean run this on an iPhone? Or do you mean run in ina Mac?

This is a python program, it is designed to run on a desktop, like a
Mac/PC/Linux. It is not meant to run on a mobile device. But I don't know,
there maybe a python version for the iPhone. In that case, should run on
it, but does not have a GUI. And I'm not that familiar with iPhone
development.

**Comment by Nephilimi** [comment]
I apologize, I thought you were the developer [behind](https://play.google.com/store/apps/details?id=org.avmedia.gshockGoogleSync) these [apps](https://f-droid.org/en/packages/org.avmedia.gshockGoogleSync/) and this was the repo mentioned in the F-Droid notes. 

iOS refers to the operating system on iPhone/iPad and not the computer MacOS. 

If you aren't familiar with apple development that's going to be a bit of a roadblock...

**Comment by izivkov** [comment, marked as answer]
No prob. Yes I'm the developer of these apps, but your message came from
the python project, so I was confused.

Unfortunately, I'm not an iOS developer, so I only implemented the Android
version. I hopysn iOS developer can implement a version for the iPhone.

Regards,
Ivo
